index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+    <header>
+       <!-- logo left -->
+        <div class="logo">
+            <h3>My Logo</h3>
+        </div>
+        <!-- strart right nav-bar -->
+        <div class="nav">
+            <i id="menu" class="material-icons">menu</i>
+        </div>
+    </header>
+<main>
+    <div class="content">
+        <h1>I’m Desmond
+            Glad to see you!
+        </h1>
+        <div class="about">
+            <h5>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </h5>
+        </div>
+    </div>
+</main>
+<footer>
+    <div class="conect">
+        <span>LET’S CONNECT</span>
+    </div>
+    <div class="icons">
+        <img src="images/Linkedinicon.jpg" alt="">
+        <img src="/images/Vectora.jpg" alt="">
+        <img src="/images/Vectorb.jpg" alt="">
+        <img src="/images/Vectorc.jpg" alt="">
+        <img src="images/Vector.jpg" alt="">
+    </div>
+</footer>
```

Comment:
- [ ] Great approach! But you will be adding more information to this project and the social icons are not the footer of the final application. Please move everything inside this footer to the `<div class='content'>` tag that you already have.
- [ ] In addition, you are required to put each social icon inside an `<li>` tag in a `<ul>` tag.